dasharo-security/secure-boot.robot

```diff
@@ -233,6 +233,52 @@ SBO008.001 Attempt to enroll the key in the incorrect format (firmware)
 
+SBO009.001 Attempt to enroll the key in the incorrect format (OS)
+    [Documentation]    This test verifies that it is impossible to load
+    ...    a certificate in the wrong file format from the OS.
+    Skip If    not ${SECURE_BOOT_SUPPORT}    SBO008.001 not supported
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    SBO008.001 not supported
+    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    SBO008.001 not supported
+    Power On
+    ${sb_menu}=    Enter Secure Boot Menu And Return Construction
+    ${advanced_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
+    Erase All Secure Boot Keys    ${advanced_menu}
+    Save Changes And Reset    3    5
+    Boot System Or From Connected Disk    ubuntu
+    Login To Linux
+    Switch To Root User
+    # remove existing directory that contains the keys
+    Execute Linux Command    rm -rf /usr/share/secureboot/keys
+    # make the directory structure
+    Execute Linux Command    mkdir /usr/share/secureboot/keys
+    Execute Linux Command    mkdir /usr/share/secureboot/keys/PK
+    Execute Linux Command    mkdir /usr/share/secureboot/keys/KEK
+    Execute Linux Command    mkdir /usr/share/secureboot/keys/db
+    Execute Linux Command    mkdir /usr/share/secureboot/keys/dbx
+    # generate and move incorrectly formatted db keys to the right place
+    Execute Linux Command
+    ...    openssl ecparam -genkey -name secp384r1 -out db.key && openssl req -new -x509 -key db.key -out db.pem -days 365 -subj "/CN=db"
+    Execute Linux Command    mv db.key /usr/share/secureboot/keys/db/
+    Execute Linux Command    mv db.pem /usr/share/secureboot/keys/db/
+    # generate and move incorrectly formatted dbx keys to the right place
+    Execute Linux Command
+    ...    openssl ecparam -genkey -name secp384r1 -out dbx.key && openssl req -new -x509 -key dbx.key -out dbx.pem -days 365 -subj "/CN=db"
+    Execute Linux Command    mv dbx.key /usr/share/secureboot/keys/dbx/
+    Execute Linux Command    mv dbx.pem /usr/share/secureboot/keys/dbx/
+    # generate and move incorrectly formatted PK keys to the right place
+    Execute Linux Command
+    ...    openssl ecparam -genkey -name secp384r1 -out PK.key && openssl req -new -x509 -key PK.key -out PK.pem -days 365 -subj "/CN=db"
+    Execute Linux Command    mv PK.key /usr/share/secureboot/keys/PK/
+    Execute Linux Command    mv PK.pem /usr/share/secureboot/keys/PK/
+    # generate and move incorrectly formatted KEK keys to the right place
+    Execute Linux Command
+    ...    openssl ecparam -genkey -name secp384r1 -out KEK.key && openssl req -new -x509 -key KEK.key -out KEK.pem -days 365 -subj "/CN=db"
+    Execute Linux Command    mv KEK.key /usr/share/secureboot/keys/KEK/
+    Execute Linux Command    mv KEK.pem /usr/share/secureboot/keys/KEK/
+    # try to enroll them, should be impossible
+    ${out}=    Execute Linux Command    sbctl enroll-keys --yes-this-might-brick-my-machine
```

Comment:
`--yes-this-might-brick-my-machine` what this does and why we are using it? Does not sound good..

Reply:
sbctl is known to brick some thinkpads when you enroll a custom PK (and remove the default one). I know because it bricked mine :D

Reply:
@TomaszAIR we have to use this flag, otherwise sbctl won't do anything